packages/core/admin/admin/src/pages/Admin/index.js

```diff
@@ -15,6 +15,7 @@ import AppLayout from '../../layouts/AppLayout';
 import { useMenu, useReleaseNotification } from '../../hooks';
 import Onboarding from './Onboarding';
 import { createRoute } from '../../utils';
+import GuidedTourModal from '../../components/GuidedTour/Modal';
```

Comment:
Would it be possible to make this a code-splitting point, by using React.lazy and dynamic imports? This way it would not increase the bundlesize of the main bundle if the feature isn't needed (non admin-users, already completed ...).